utils.groovy

```diff
@@ -384,6 +384,12 @@ def streams_of_type(config, type) {
     return config.streams.findAll{k, v -> v.type == type}.collect{k, v -> k}
 }
 
+// Returns a list of stream names from `streams_subset` that have `scheduled: true` set
+def scheduled_streams(config, streams_subset) {
+    return streams_subset.findAll{stream ->
+        config.streams[stream].scheduled == true}.collect{k, v -> k}
```

Comment:
```suggestion
        config.streams[stream].scheduled}.collect{k, v -> k}
```

Reply:
Thanks! I'll make that change

Reply:
Looking at this again, shouldn't this be kept as `.scheduled == true` to collect only the mechanical streams that have `scheduled: true`?

Reply:
Hmm, not sure I follow. It should evaluate to false whether it's unset or set explicitly to `false`.

Reply:
I think you did. You explained why it doesn't need to be kept as I wrote it. 

Thanks! I suppose I was trying to be explicit here, but I see groovy still evaluates to true in this case. I'll update the PR with all the suggestions.